react-list.js

```diff
@@ -145,10 +146,10 @@
     _createClass(ReactList, [{
       key: 'componentWillReceiveProps',
       value: function componentWillReceiveProps(next) {
-        var _state = this.state,
```

Comment:
@ibash this changes came from master. Are we somehow using different lint version/config?